mdf_matio/__init__.py

```diff
@@ -103,13 +103,67 @@ def is_in_directory(f):
         yield _merge_records(group)
 
 
-def generate_search_index(directory: str, validate_records=True,
+def _call_xtracthub(data_url: str, index_options: dict, target_parsers: Set[str]) -> Iterable[ParseResult]:
```

Comment:
Sorry, I know what I'm asking will probably be a pain in the butt.

Reply:
It appears that a rebase didn't have the desired effect. Is this the only change from `xtracthub` that you want removed? If so, I can just delete this function manually. Otherwise I'll need to look more into how to fully separate `adapter-class` from `xtracthub` cleanly.

Reply:
Sure, feel free to delete it manually. Seems like an easy fix to another otherwise complicated dance with `git`